### PR TITLE
Fix runtime not being python

### PR DESCRIPTION
**Pull request by mklenbw**
When runtime is not set to python don't use it as pythonBin.

Closes: #741

### CODE REVIEW

**Comment by pgrzesik** [comment]
Thanks @mklenbw - sorry for not responding sooner, but I've been very busy lately, I'll try to review this PR by the end of the week

**Review by pgrzesik — COMMENTED** [review]
Hey @mklenbw, thanks a lot for your patience, I'm really sorry that I wasn't able to respond sooner to your PR. Do you think you could introduce a test case that covers this case? Please let me know what do you think 🙇

**Comment by mklenbw** [comment]
Sure. I'll add it right away and text you.

**Comment by ShantanuNair** [comment]
+1 Also facing this issue. Appreciate the PR!

**Comment by mklenbw** [comment]
@pgrzesik 
Sorry i forgot to push it. Now it's there. Hope this helps.

**Comment by pgrzesik** [comment]
Hey @mklenbw, thank you so much 🙇 Unforutnatelly it looks like formatting is failing, could you please check that?

**Comment by mklenbw** [comment]
@pgrzesik Got it fixed. Forgot to run prettier. You should consider including husky. 
There's another problem with the unit-tests. I guess it was the node16.x runtime (might fail on node14 and node12).
The node12-test should be removed as it's EOL since 30.04.2022.

File: test.js [hunk 1]
```diff
@@ -1655,6 +1655,16 @@ test('poetry py3.7 fails packaging if poetry.lock is missing and flag requirePoe
   t.end();
 });
 
+test('works with provider.runtime not being python', async (t) => {
+  console.log('im working');
```
Comment (pgrzesik) [inline test.js:1659]: You might want to remove that `console.log` here as it might be messing up the test

**Comment by pgrzesik** [comment]
Hey @mklenbw - thank you, there's a leftover `console.log` that might be messing up the tests. As for Node12 - modernize the CI matrix on my list of things to do soon 👍

**Comment by mklenbw** [comment]
@pgrzesik 
Oh sorry. Never worked with tape before and needed some visuals ;-). Thanks for looking around.

**Comment by mklenbw** [comment]
@pgrzesik 
I'm sorry. I really tried fixing this, but the most test-requirements are so old that my company doesn't allow installation or requirements like perl are just undocumented and unnecessary, that i'm wasting to much time on this.
That means i will quit working on this PR and just take the plugin as broken and outdated as it is.

If someone wants to continue this topic please take my code and get it to work. I would love to see this masterpiece of a plugin in it's full glance again.

**Comment by pgrzesik** [comment]
No worries, sorry that it became an issue and thanks for all the effors @mklenbw 👍 I'll try to wrap this up over the weekend. I'm trying to keep the lights on with this particular plugin but unfortunately my time has been very limited lately.

**Comment by harry-isaac** [comment]
+1 for would love to see this fixed

**Comment by jameskbride** [comment]
Picked up where @mklenbw left off, see #773 .

**Comment by pgrzesik** [comment]
thanks @jameskbride - I'll close this one and respond in other PR

**Pull request closed without merging**